<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HomePage</class>
 <widget class="QMainWindow" name="HomePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>333</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="page_SW1">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QCalendarWidget" name="calendarWidget"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_SW2">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLCDNumber" name="lcdNumber"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_SW3">
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QTextBrowser" name="textBrowser"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>470</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionHalaman_1"/>
    <addaction name="separator"/>
    <addaction name="actionHalaman_2"/>
    <addaction name="separator"/>
    <addaction name="actionHalaman_3"/>
    <addaction name="separator"/>
    <addaction name="actionHalaman_4"/>
    <addaction name="separator"/>
    <addaction name="actionMap_3"/>
    <addaction name="separator"/>
    <addaction name="actionMap_4"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="menuExit"/>
   </widget>
   <addaction name="menuMenu"/>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionHalaman_1">
   <property name="text">
    <string>Halaman 1</string>
   </property>
  </action>
  <action name="actionHalaman_2">
   <property name="text">
    <string>Halaman 2</string>
   </property>
  </action>
  <action name="actionHalaman_3">
   <property name="text">
    <string>Map 1</string>
   </property>
  </action>
  <action name="menuExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionHalaman_4">
   <property name="text">
    <string>Map 2</string>
   </property>
  </action>
  <action name="actionMap_3">
   <property name="text">
    <string>Map 3</string>
   </property>
  </action>
  <action name="actionMap_4">
   <property name="text">
    <string>Map 4</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
